SMLHelper/Initializer.cs

```diff
@@ -32,6 +35,15 @@ public static void PrePatch()
             TechTypePatcher.cacheManager.LoadCache();
             Logger.Debug("Loading CraftTreeType Cache");
             CraftTreeTypePatcher.cacheManager.LoadCache();
+
+            try
+            {
+                PreInitialize();
+            }
+            catch(Exception e)
+            {
+                Logger.Error($"Caught exception while trying to initialize SMLHelper{Environment.NewLine}{e}");
```

Comment:
Remove the try-catch.
Let QModManager handle any exceptions that happen during patching.